src/Lime.Transport.WebSocket/ClientWebSocketTransport.cs

```diff
@@ -63,11 +63,18 @@ public override string LocalEndPoint
                 }
                 catch
                 {
-                    return base.LocalEndPoint;
+                    try
+                    {
+                        return WebSocket.AsDynamic()._innerWebSocket?.WebSocket?._stream?._connection?._socket?.LocalEndPoint?.ToString();
```

Comment:
This is the '.Net 6 version', right? I think it could be the first one here.

Reply:
Done.